dqc_us_rules/dqc_us_0015.py

```diff
@@ -72,7 +73,7 @@ def filter_negative_number_facts(val, blacklist_concepts):
     # other filters before running negative numbers check
     # numeric_facts has already checked if fact.value can be made into a number
     facts_to_check = [
-        f for f in numeric_facts if float(f.value) < 0 and
+        f for f in numeric_facts if f.xValid >= VALID and f.xValue < 0 and
```

Comment:
Using xValid will give us "at minimum, consistency with other code.  I (Herm) is not sure if xValue is set or what it might be if not valid"